kordac/Kordac.py

```diff
@@ -2,22 +2,35 @@
 import mdx_math
 from kordac.KordacExtension import KordacExtension
 
-class Kordac():
+class Kordac(object):
+    """A converter object for converting markdown
+    to HTML"""
 
-    def run(self, md_string):
-        self.heading = 'I am a heading'
+    def run(self, markdown_string):
+        kordac_extension = KordacExtension()
+        kordac_extension.heading = None
         self.required_files = {}
-        self.html_string = ''
-        html = None
-        ext = KordacExtension()
+
         converter = markdown.Markdown(extensions=[
```

Comment:
seeing as you've already done some refactoring here, in this same PR you might as well move the converter to be in the `__init__()` function (as you suggested earlier)